packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue

```diff
@@ -282,6 +283,14 @@ export default {
       default: true,
     },
 
+    /**
+     * Whether the input allows inline images to be rendered.
+     */
+    allowInlineImages: {
+      type: Boolean,
+      default: true,
+    },
```

Comment:
We should set this prop to false in the message_input component for now since the backend is not likely able to handle them (hopefully it will after my rich text launch-a-thon!)